sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -243,7 +243,7 @@ partitionSpec
     ;
 
 partitionVal
-    : identifier (EQ constant)?
+    : identifier (comparisonOperator constant)?
```

Comment:
You mean that we change this as an expression and validate differently case by case.

For example,
- `ADD PARTITION` should validate `column=value`.
- `DROP PARTITION` should validate `column comparisonOperator constant` except NSEQ.

Did I understand correctly?